### ISSUE TITLE
Libuna ebuild issue.

### ISSUE THREAD

**Issue by vandr0iy** [Spike-Pentesting/spike-overlay#27]
Hi, spike devs!
I've found an issue concerning the ebuild of libuna, which is a dependency of Digital Forensics Framework. 
Here is the output of "emerge -a dff"
http://pastebin.com/V19cjTWp

**Comment by mudler** [comment]
Hi, thanks for dropping by,

the package you are trying to install it's not in this overlay, but is from spike-community-overlay which is not managed by us.

Cheers

**Comment by vandr0iy** [comment]
I shall open this issue in their page too. Anyways, libuna ebuild is present here too, and has the same exact problem. I tried to install it both from here and from spike-community-overlay (by enabling/disabling respective overlays in layman), and got the same output. 
The issue consists in the fact that the download folder appears to be specified incorrectly, causing the emerge to fail in the fetch fase. I solved this problem by setting up a local overlay, hardcoding the date in the download path, and creating an empty folder in my_local_overlay/dev-libs/libuna-20150927 (which is the date I had to hardcode). Only THEN it worked more or less correctly. 

Thank you for replying.
